SwedbankPaySDK/Classes/SwedbankPaySDK+Extensions/ViewConsumerIdentificationInfo.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright 2020 Swedbank AB
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+public extension SwedbankPaySDK {
+    /// Data required to show the checkin view.
+    ///
+    /// If you provide a custom SwedbankPayConfiguration
+    /// you must get the relevant data from your services
+    /// and supply a ViewConsumerIdentificationInfo
+    /// in your SwedbankPayConfiguration.postConsumers
+    /// completion call.
+    struct ViewConsumerIdentificationInfo {
+        /// The url to use as the WKWebView page url
+        /// when showing the checkin UI.
+        var webViewBaseURL: URL?
+        
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)